ControlRoom/Controllers/SimCtl+SubCommands.swift

```diff
@@ -0,0 +1,694 @@
+//
+//  SimCtl+SubCommands.swift
+//  ControlRoom
+//
+//  Created by Patrick Luddy on 2/16/20.
+//  Copyright © 2020 Paul Hudson. All rights reserved.
+//
+
+import Foundation
+
+// swiftlint:disable file_length
+extension SimCtl {
+    enum SubCommand {
+        /// Create a new device.
+        case create(name: String, deviceTypeId: String, runtimeId: String? = nil)
+        /// Clone an existing device.
+        case clone(deviceId: String, name: String)
```

Comment:
`simctl` docs call this `device`, but I think all of our usages call this `simulator`.  Maybe this should align to our language in other places and be `clone(simulator: String ...` etc?